http/src/test/java/org/mule/test/module/http/functional/listener/HttpListenerPersistentConnections10TestCase.java

```diff
@@ -50,8 +50,7 @@ public void persistentClosePropertyCheckHeader() throws Exception {
 
   @Test
   public void persistentEchoCheckHeader() throws IOException {
-    // Echo sets the content-lenght at 0, so keep-alive is ok for 1.0
-    assertThat(performRequest(persistentStreamingPort.getNumber(), getHttpVersion(), true), is(KEEP_ALIVE));
+    assertThat(performRequest(persistentStreamingPort.getNumber(), getHttpVersion(), true), is(CLOSE));
```

Comment:
The comment is not valid anymore since null is not propagated (which caused the 0 content-length). Under those circumstances it seems the behaviour should be a `close`.